docker-compose.ci.env

```diff
@@ -58,37 +58,35 @@ STATE_COMMITMENT_CHAIN_CONTRACT_ADDRESS=0x3A098b2982EC25772f16C57f2C1D0463E052ec
 L1_TO_L2_TRANSACTION_QUEUE_CONTRACT_ADDRESS=0xA1B22bF35196AFCE0927D94ce8ad4C4b7bb6F005      # (Required) The address of the L1ToL2TransactionQueue contract
 SAFETY_TRANSACTION_QUEUE_CONTRACT_ADDRESS=0x3a3BEFB4942cF20A69C8eb4FDB8957223Da57fa2        # (Required) The address of the SafetyTransactionQueue contract
 L1_CHAIN_DATA_PERSISTER_DB_PATH=/mnt/l1-node                                                # (Required) The filepath where to locate (or create) the L1 Chain Data Persister LevelDB database
-L1_EARLIEST_BLOCK=0                                                                   # (Required) The earliest block to sync on L1 to start persisting data
```

Comment:
Oh @willmeister I might have forgotten to mention this! @tynes & I discovered that if variables in the `env` files are set to:

```
ENV_VAR=                                                 # Some comment
```

The env variable seems to actually be getting set to the *comment*!!!! This is super scary and could cause some very hard to track down bugs. I don't see any instances in this env file but thought I'd mention it because it could cause grief down the line